meta/recipes-kernel/perf/perf.bb

```diff
@@ -81,8 +81,8 @@ EXTRA_OEMAKE = '\
     LDSHARED="${CC} -shared" \
     AR="${AR}" \
     LD="${LD}" \
-    EXTRA_CFLAGS="-ldw" \
-    YFLAGS='-y --file-prefix-map=${WORKDIR}=/usr/src/debug/${PN}/${EXTENDPE}${PV}-${PR}' \
+    EXTRA_CFLAGS="-ldw -Wno-array-bounds" \
```

Comment:
I need some more commentary here about why it is appropriate to ignore these warnings. Is this fixed in the `perf` upstream?

Reply:
Basically we are stuck on an old kernel, so instead of pulling in fixes to the kernel for this, we just ignore the warnings like the previous compiler did.

Reply:
https://github.com/ni/openembedded-core/pull/68/commits/75611681599427339562980bb974c02ba1ca62f9

Could you add that justification to the commit message and sign-off the commit?

Reply:
Done.